float32 angle_to_spin
---
uint32 return_message
---
float32 current_angle
